src/darwin/Framework/CHIP/CHIPDeviceController.h

```diff
@@ -115,9 +115,10 @@ typedef void (^CHIPDeviceConnectionCallback)(CHIPDevice * _Nullable device, NSEr
 + (instancetype)new NS_UNAVAILABLE;
 
 /**
- * Return the Node Id assigned to the controller.
+ * Return the Node Id assigned to the controller.  Will return nil if the
+ * controller is not running (and hence does not know its node id).
  */
-- (NSNumber *)getControllerNodeId;
+- (nullable NSNumber *)getControllerNodeId;
```

Comment:
nit pick, do we make this a readonly property?

Reply:
Done.